### PR TITLE
feat(sidebar) add tabindex and keyboard navigation to sidebar

### PR DESCRIPTION
**Pull request by Mierenga**
Sidebar is now accessible through the tab index. You can also use `<Enter>`, `<ArrowRight>`, and `<ArrowLeft>` to open and close sections and use `<Enter>` to select an item.

### CODE REVIEW

**Review by nateslo — APPROVED** [review]
Manually tested, LGTM

File: src/components/SidebarList.js [hunk 1]
```diff
@@ -112,17 +149,29 @@ export default class SidebarList extends React.Component {
...
+                onKeyUp={(e) => this.subMenuKeyup(e.key, tag)}
+              >
+                {tag}
+              </span>
+              <ul className="submenu-items" role="menu">
                 {sidebarItem.get("operations").map(op =>
                   <div>
-                    <li className={"method " + this.ifActive(this.isIdActive(op.get("id")))} >
-                      <a onClick={() => this.sidebarAnchorClicked(tag, op)} className={"method-" + op.get("method")}>
+                    <li tabIndex={this.isTagActive(tag) ? 0 : -1}
```
Comment (nateslo) [inline src/components/SidebarList.js:167]: non-blocking nitpick/question:  should `tabIndex` be on a new line? There seems to be some variation in styling, not just in this file, but also in other places throughout Swagger-ui.

File: src/components/SidebarList.js [hunk 2]
```diff
@@ -112,17 +149,29 @@ export default class SidebarList extends React.Component {
...
+                      role="menuitem"
+                      className={"method " + this.ifActive(this.isIdActive(op.get("id")))}
+                      onKeyUp={(e) => this.sidebarAnchorKeyup(e.key, tag, op)}
+                    >
+                      <a onClick={() => this.sidebarAnchorClicked(tag, op)}
```
Comment (nateslo) [inline src/components/SidebarList.js:172]: same comment as above

File: src/components/SidebarList.js (hunk 1 above)
Comment (Mierenga) [reply src/components/SidebarList.js:167]: I've definitely seen my share of both styles in different projects. I think starting attributes that need to wrap around on the next line does look a little cleaner. Happy to go with that as the way towards consistency.

**Pull request merged**